### PR TITLE
Add Rackspace CDN to the landing page

### PR DESCRIPTION
**Pull request by smashwilson**
This is a resubmission of @cyrichardson's pull requests from last week: #1070, #1069, #1068.

### CODE REVIEW

**Comment by smashwilson** [comment]
@cyrichardson: Would you mind looking this over and making sure I didn't miss any changes?

Also, it looks like dev and master are a little out of sync right now: specifically, the regions page has been added to master, but never merged to dev. I'll submit another PR to bring dev up to date, and touch this up if necessary once it makes its way in.

File: src/site_source/_includes/docs-sidebar.html [hunk 1]
```diff
@@ -14,6 +14,7 @@
   <li><a href="/docs/cloud-monitoring/getting-started/">Cloud Monitoring</a></li>
   <li><a href="/docs/cloud-queues/getting-started/">Cloud Queues</a></li>
   <li><a href="/docs/cloud-servers/getting-started/">Cloud Servers</a></li>
+  <li><a href="/docs/rackspace-cdn/getting-started/">Rackspace CDN</a></li>
```
Comment (ycombinator) [inline src/site_source/_includes/docs-sidebar.html]: Please remove this change as the examples for all SDKs are not ready yet.

**Comment by smashwilson** [comment]
.Rebased on top of the updated `dev`. @ycombinator's comment still stands.

File: src/docs/index.rst [hunk 2]
```diff
@@ -20,10 +20,10 @@
                     <img src="{% asset_path cloud-servers.png %}" alt="" class="product-image"/>
                     Auto Scale
                 </h2>
-                <p>Rackspace Auto Scale is a service that lets you configure automated scaling of resources in response to an 
+                <p>Rackspace Auto Scale is a service that lets you configure automated scaling of resources in response to an
```
Comment (smashwilson) [inline src/docs/index.rst]: These whitespace changes were me - sorry. They shouldn't hurt anything, though.

File: src/docs/index.rst [hunk 3]
```diff
@@ -190,16 +190,16 @@
...
         </div>
 
-      <div class="row">
+        <div class="row">
             <div class="col-md-12">
                 <h2>
-                  <img src="{% asset_path cloud-identity.png %}" alt="" class="product-image"/>
-                  Cloud Identity
+                  <img src="{% asset_path icon-cdn.png %}" alt="" class="product-image"/>
+                  Rackspace CDN
```
Comment (ycombinator) [inline src/docs/index.rst]: These set of changes look like they are _replacing_ Cloud Identity content with Rackspace CDN content on this page. Is that what we want?

Comment (smashwilson) [inline src/docs/index.rst]: Oh, weird. That's probably my fault. I think Cloud Identity had been added or removed in either dev or master and I must have gotten confused resolving the conflicts.

**Comment by smashwilson** [comment]
> So Rackspace CDN really can't be added to this page until the SDK work is done. Looking at Cloud Identity in the right hand pane (only api ref and dev guide), I was thinking I could do that for Rackspace CDN. And the SDK quick starts could be added later.
>
> So Ash, I guess we close the PR again and wait until 3/25.

:bow: Aye aye!

**Pull request closed without merging**